javascript/src/main/components/Profile/RoleBadge.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Badge } from "react-bootstrap";
+
+export default function RoleBadge(props) {
+    const {roleInfo} = props;
+  return (
```

Comment:
The spacing appears to be slightly inconsistent here (not sure if it’s just me reviewing this on my phone though). Small nitpick.